maven-core/src/main/resources/META-INF/plexus/components.xml

```diff
@@ -101,10 +101,10 @@ under the License.
         </phases>
         <default-phases>
           <site>
-            org.apache.maven.plugins:maven-site-plugin:3.3:site
+            org.apache.maven.plugins:maven-site-plugin:3.9.1:site
```

Comment:
This is the latest version in the [Maven Central Repository](https://search.maven.org/artifact/org.apache.maven.plugins/maven-site-plugin).